internal/pfcp/pfcp.go

```diff
@@ -58,6 +58,7 @@ type PfcpServer struct {
 	rxTrans      map[string]*RxTransaction // key: RemoteAddr-Sequence
 	txSeq        uint32
 	log          *logrus.Entry
+	urrURSEQN    map[uint32]uint32
```

Comment:
what is `urrURSEQN` used for? It is not used in this PR. If no more use, please remove it.